src/feature/hs/hs_config.c

```diff
@@ -444,6 +444,11 @@ config_service_v3(const config_line_t *line_,
   return -1;
 }
 
+/**
+ * DOCDOC
+ **/
```

Comment:
If you have the chance here to maybe add the comment about this. Seems pretty simple, that this is the section header to configure an onion service.